src/key.js

```diff
@@ -1489,3 +1501,45 @@ export async function getPreferredSymAlgo(keys) {
   }
   return prefAlgo.algo;
 }
+
+/**
+ * Returns the preferred aead algorithm for a set of keys
+ * @param  {Array<module:key.Key>} keys Set of keys
+ * @param  {Date} date (optional) use the given date for verification instead of the current time
+ * @returns {Promise<module:enums.aead>} Preferred aead algorithm, or null if the public keys do not support aead
+ * @async
+ */
+export async function getPreferredAeadAlgo(keys, date) {
```

Comment:
Can we combine this with the getPreferredSymAlgo function? Lots of duplicated code